src/approaches/base_approach.py

```diff
@@ -64,16 +66,27 @@ def seed(self, seed: int) -> None:
     def learn_from_offline_dataset(self, dataset: Dataset) -> None:
         """For learning-based approaches, learn whatever is needed from the
         given dataset. Also, save whatever is necessary to load() later.
-
-        Note: this is not an abc.abstractmethod because it does
-        not need to be defined by the subclasses. (mypy complains
-        if you try to instantiate a subclass with an undefined abc).
         """
 
     def load(self) -> None:
         """Load anything from CFG.get_approach_save_path_str().
 
-        Only called if self.is_learning_based.
+        Only called if self.is_learning_based, BEFORE any online learning.
```

Comment:
what do you mean by this exactly?

Reply:
right now only load_from_offline_dataset() will do any saving, and then load() will load the result of that, before starting the learning loop. i'll clarify the comment. later on, we can fix this so that saving/loading can happen online too

Reply:
updated the doc

Reply:
okay sounds good. can we make an issue so we don't forget? :)

Reply:
okay once we merge this PR